packages/dd-trace/src/proxy.js

```diff
@@ -155,6 +156,18 @@ class Tracer extends NoopProxy {
         if (config.dynamicInstrumentation.enabled) {
           DynamicInstrumentation.start(config, rc)
         }
+
+        if (config.flaggingProvider.enabled) {
+          rc.setProductHandler('FFE_FLAGS', (action, conf) => {
+            // Feed UFC config directly to flagging provider
+            if (action === 'apply' || action === 'modify') {
+              this.flaggingProvider._setConfiguration(conf.flag_configuration)
+            } else if (action === 'unapply') {
+              // For now, just pass empty config
+              this.flaggingProvider._setConfiguration({})
```

Comment:
Can you pass `this.flaggingProvider._setConfiguration(undefined)` instead? Passing an empty object would cause an `undefined is not an object` error during evaluation since it expects a "flags" key to exist.

Reply:
I'am going to remove the handling of that action for now since we don't have a defined spec for it, could add it later as need be